string command
---
float64[] state
float64 reward
bool done
